xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd

```diff
@@ -490,19 +489,19 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element ref="TypeOfProductCategoryRef" minOccurs="0"/>
 			<xsd:element ref="TypeOfServiceRef" minOccurs="0"/>
-			<xsd:element name="UserTypes" type="UserTypeListOfEnumerations" minOccurs="0">
+			<xsd:element name="AccessModes" type="AccessModeListOfEnumerations" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>If only specific user types are allowed on the line, they can be indicated in this list.</xsd:documentation>
+					<xsd:documentation>List of ACCESS MODEs that are allowed for this line. If not specified means all the usual ones for classic public transport. Can be used for example to indicate in vehicle only access. +v2.0</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="AccessModes" type="AccessModeListOfEnumerations" minOccurs="0">
+			<xsd:element name="RestrictedLine" type="xsd:boolean" default="false" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>List of ACCESS MODEs that are allowed on this line. Omitted means the "usual" ones on classic public transport.</xsd:documentation>
+					<xsd:documentation>Whether the LINE is restricted to special groups of passengers and not open to the general public. Default is false. +v2.0</xsd:documentation>
```

Comment:
In OJP, RestrictedLine not only considers (the analog to) UserTypes, but also AccessModes. Shouldn't we take the same approach here? I.e., "Whether [....], or restricted by the access modes supported." ?

Reply:
included